include/ipc/ipc_service_backend.h

```diff
@@ -71,6 +71,63 @@ struct ipc_service_backend {
 	int (*register_endpoint)(const struct device *instance,
 				 void **token,
 				 const struct ipc_ept_cfg *cfg);
+
+	/** @brief Pointer to the function that will return an empty TX buffer.
+	 *
+	 *  @param instance Instance pointer.
+	 *  @param token Backend-specific token.
+	 *  @param data Pointer to the empty TX buffer.
+	 *  @param len Pointer to store the TX buffer size.
+	 *  @param wait To wait or not for the buffer to become available.
```

Comment:
this cannot be a simple boolean - let's use timeout for that

Reply:
ok